docs/content/en/docs/concepts/performance-stats.md

```diff
@@ -0,0 +1,172 @@
+---
+title: "BPF programs statistics"
+weight: 5
+description: "Monitor bpf programs statistics"
+---
+
+This page shows you how to monitor bpf programs statistics.
+
+## Concept
+
+The BPF subsystem provides performance data for each loaded program and tetragon
+exports that in metrics or display that in terminal in top like tool.
+
+## In terminal
+
+The tetra command allows to display loaded BPF programs in terminal with:
+
+```
+# tetra debug progs
+```
```

Comment:
````suggestion
```shell
tetra debug progs
```
````

Reply:
Similarly for all the other commands with `#`, please remove it and put the `shell` text above to show it's cmd.